deis-lite/manifests/deis-builder-rc.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: deis-builder
+  namespace: deis
+  labels:
+    heritage: deis
+spec:
+  replicas: 1
+  selector:
+    name: deis-builder
```

Comment:
It'd be good to use `app` for selectors. I'm in the process of moving that way in the other chart as well: https://github.com/deis/charts/pull/41